boards/px4/fmu-v5/src/manifest.c

```diff
@@ -160,6 +160,8 @@ static px4_hw_mft_list_entry_t mft_lists[] = {
 	{0x0000, hw_mft_list_v0500,        arraySize(hw_mft_list_v0500)},
 	{0x0105, hw_mft_list_v0500,        arraySize(hw_mft_list_v0500)},  // Alias for CUAV V5 R:5 V:1
 	{0x0500, hw_mft_list_v0500,        arraySize(hw_mft_list_v0500)},  // Alias for CUAV V5+ R:0 V:5
+	{0x0502, hw_mft_list_v0500,        arraySize(hw_mft_list_v0500)},  // Alias for CUAV V5+ R:2 V:5 ICM42688P
+	{0x0602, hw_mft_list_v0500,        arraySize(hw_mft_list_v0500)},  // Alias for CUAV V5nano R:2 V:6 ICM42688P
```

Comment:
What about the entries for V552 and V562?

Reply:
According to the macro definition: define HW_VER_REV(v,r) ((uint32_t)((v) & 0xff) << 8) | ((uint32_t)(r) & 0xff), 0x0502 is V552, 0x0602 is V562.

Reply:
right! Would you mind adding and using the format from 

https://github.com/PX4/PX4-Autopilot/blob/master/boards/px4/fmu-v5x/src/board_config.h#L182-L199

So this will no be confused again.

Reply:
I have modified. Please review.